src/index.html

```diff
@@ -1,14 +1,60 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Menu Example</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img
+        src="images/logo.png"
+        alt="Logo"
+        class="logo"
+        >
```

Comment:
1. unused class logo, you don't use it for styling
2. less spacings
```suggestion
        <img
          src="images/logo.png"
          alt="Logo"
          class="logo"
        >
```
as class is unused it's better to make it this way:
```suggestion
        <img src="images/logo.png" alt="Logo">
```
3. alt should be more describable, it can contain name of the project `moyo logo`